src/test/spec/unified_runner/test_runner.rs

```diff
@@ -128,6 +129,14 @@ impl TestRunner {
         ));
 
         for test_case in test_file.tests {
+            // to run a single test, set the TEST_DESCRIPTION environment variable to the test's
+            // description
+            if let Ok(description) = std::env::var("TEST_DESCRIPTION") {
+                if description != test_case.description {
+                    continue;
+                }
+            }
```

Comment:
I think we all probably write something to this effect when trying to debug an individual test, so I thought it would be useful to have this in the code permanently

Reply:
Could we make this `contains` instead of strict equality? I normally add something like that in case I want to run a few tests at once. Great idea to add this btw; I think it'll save a ton of time for all of us going forward.

Reply:
nice 😎